frontend_tests/node_tests/composebox_typeahead.js

```diff
@@ -84,6 +84,8 @@ global.people.add({
     assert_typeahead_equals("hi emoji :", false);
     assert_typeahead_equals("hi emoji :ta", emoji_list);
     assert_typeahead_equals("hi emoji :da", emoji_list);
+    assert_typeahead_equals("hi emoji :da_", emoji_list);
+    assert_typeahead_equals("hi emoji :da ", emoji_list);
```

Comment:
I'm a little confused here; how can "hi :see no" and "hi emoji :da_" match the same emoji_list?